pages/docs/tutorials/native/dynamic-libraries.md

```diff
@@ -187,26 +207,34 @@ all functions that Kotlin/Native and our library provides to us. It heavily uses
 nested anonymous structures to mimic packages.
 
 The `demo_ExportedSymbols` structure contains several helper functions:
+
+<div class="sample" markdown="1" mode="C" theme="idea" data-highlight-only="1" auto-indent="false">
+
 ```c
 void (*DisposeStablePointer)(demo_KNativePtr ptr);
 void (*DisposeString)(const char* string);
 demo_KBoolean (*IsInstance)(demo_KNativePtr ref, const demo_KType* type);
 ```
+</div>
 
 Those functions deals with Kotlin/Native objects. One calls 
 `DisposeStablePointer` to release a Kotlin object and `DisposeString` to release Kotlin String, 
 which has the `char*` type in C. One may use the `IsInstance` function to check, if a
-Kotlin type, a `demo_KNativePtr` is an instance of another type. 
+Kotlin type, a `demo_KNativePtr` is an instance of another type. The actual set of
+operations generated depend on the actual usage from your code.
  
 Kotlin/Native has garbage collection, but it does not help to deal
 with Kotlin objects from C language. Kotlin/Native has an interop with Objective-C and 
 Swift, and integrates with their reference counters. You may want to find more details
 in the [documentation](https://github.com/JetBrains/kotlin-native/blob/master/OBJC_INTEROP.md)
 or to check the related [TUTORIAL_KOTLIN_macOS_FRAMEWORK_LINK].
 
+NOTE. It is possible to have several Kotlin/Native dynamic libraries in one application, 
```

Comment:
Does it really work on macOS?

Reply:
removed that line for now. It is hard to test, but looks possible